api/admin/routes.py

```diff
@@ -906,6 +950,57 @@ def refresh(identifier_type, identifier):
 @requires_admin
 @requires_csrf_token
 def resolve_complaints(identifier_type, identifier):
+    """Resolve all complaints for a particular license pool and complaint type.
+    ---
+    post:
+      tags:
+        - works
+      summary: Resolve all complaints for a particular license pool and complaint type.
+      security:
+        - BasicAuth: []
+      parameters:
+        - X-CSRF-Token
+        - in: path
+          name: library_short_name
+          schema:
+            type: string
+          description: The short code of a library that holds the requested work
+        - in: path
+          name: identifier_type
+          schema:
+            type: string
+          description: The type of the identifier being used to retrieve a work
+        - in: path
+          name: identifier
+          schema:
+            type: string
+          description: An identifier for a work record
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema: 
+              type: string
+              name: type
```

Comment:
What's required to go here? If not comprehensive an example would be really helpful.